x-pack/plugin/core/src/main/resources/logs-mappings.json

```diff
@@ -3,51 +3,23 @@
     "mappings": {
       "dynamic_templates": [
         {
-          "strings_as_keyword": {
+          "match_message": {
+            "match_mapping_type": "string",
+            "match": "message",
             "mapping": {
-              "ignore_above": 1024,
-              "type": "keyword"
-            },
-            "match_mapping_type": "string"
+              "type": "match_only_text"
```

Comment:
Is this a breaking change?

Reply:
What aspect are you concerned with? I expect this to be transparent for the vast majority of our users, and I started discussions with the ECS team to make similar changes on ECS.

Reply:
Read up on it by now a bit more. What happens if a user runs a non "supported" query across `logs-*` and some message fields are `text` and some are `match_only_text`. Is an error returned or the non supported fields are just skipped?

Reply:
The only unsupported queries are `span` queries. If a span query is run, then all shards that have the field mapped as `match_only_text` will be ignored from the search response and reported as "failed".